superset-frontend/cypress-base/cypress/integration/dashboard/utils.ts

```diff
@@ -195,9 +195,9 @@ export function expandFilterOnLeftPanel() {
 }
 
 /** ************************************************************************
- * Collapes Native Filter from the left panel on dashboard
+ * Collapses Native Filter from the left panel on dashboard
  * @returns {None}
- * @summary helper for collape native filter
+ * @summary helper for collapse native filter
```

Comment:
collapsing?

Reply:
Dunno. It's also possible the one above just wants `Collapse` and not `Collapses`. It depends on whether it's a proper noun or prose. If it's propose, the `for` might be best change to `to`.

Reply:
I've made a series of additional changes to this file. Including fixing another pair of typos -- https://github.com/jsoref/superset/commit/6fe4623925df39ce5defd96d54e943ac835d39ab.

Note that there are [~3k word-like tokens that I just accepted](https://github.com/check-spelling/superset/blob/13157cd3009b83897f034735b17bc1cad592191c/.github/actions/spelling/expect.txt) -- for smaller projects, I'll review each one, but for larger projects, I aim towards incremental improvement as opposed to absolute perfection :). I do encourage others to browse through them and see if any of the ones I skipped are also errors.